students/Snayxts/module-2-html/exercises/zdk09/products.html

```diff
@@ -0,0 +1,81 @@
+        </svg>
+
+        <h1>How we can help</h1>
+
+        <nav>
+            <a href="index.html">Home</a><br>
+            <a href="About.html">About</a><br>
+            <a href="#gitt">Get in contact</a>
+        </nav>
+    </header>
+
+    <article>
+        <header>
+            <h2>Smartphones and tablets</h2>
+        </header>
+
+        <section>
+            <img src="slike/product-1.jpg" alt="Phone and tablet">
+
+            <p>Marketing branding backing user experience learning curve partner network long tail seed money bandwidth venture churn rate traction crowdsource. Release vesting period backing interaction design early adopters. Ownership branding business model canvas return on investment backing android freemium series A financing holy grail facebook value proposition hypotheses entrepreneur network effects. Return on investment channels prototype bandwidth.</p>
+        </section>
+        
+        <section>
+            <ul>
+                <li>Early adopters incubator agile development disruptive deployment paradigm shift pitch founders</li>
+                <li>Hackathon facebook creative first mover advantage disruptive deployment stock equity churn rate twitter launch party</li>
+                <li>Business plan marketing long tail innovator funding android seed round</li>
+                <li>IPad paradigm shift branding seed money</li>
+            </ul>
+        </section>
+    </article>
+
+    <article>
+        <header>
+            <h2>Smart watches</h2>
+        </header>
+
+        <section>
+            <img src="slike/product-2.jpg" alt="Smart watch">
+
+            <p>Lean startup agile development stealth deployment pitch iPad iPhone assets. Mass market vesting period first mover advantage analytics creative. Burn rate bandwidth creative conversion traction gen-z niche market disruptive business plan. Entrepreneur disruptive business-to-business partner network.</p>

```

Comment:
Paragraf i popis mogu pripadati istoj sekciji jer ne postoji jasan razlog zašto bi se razdvojili.